code/game/machinery/reagents/pump.dm

```diff
@@ -138,25 +145,25 @@
 	return FALSE
 
 
-// Returns 0 on failure and 1 on success
+// Returns FALSE on failure and TRUE on success
 /obj/machinery/pump/proc/turn_on(var/loud = 0)
 	if(!cell)
-		return 0
-	if(cell.charge < (use * CELLRATE))
-		return 0
+		return FALSE
+	if(!cell.checked_use(200))
```

Comment:
Why swap this out for a magic number?

Reply:
Was doing testing and swapped for the bare use number, forgot to swap back.